src/mm2_common.h

```diff
@@ -25,12 +25,15 @@ Multiple declarations will cause compiler errors!
 // Allocator
 // Calls MM2's internal operator new
 //
-#define ANGEL_ALLOCATOR void* operator new(size_t size)          { return hook::StaticThunk<0x577360>::Call<void*>(size); } \
-                        void operator delete(void * pointer)     { hook::StaticThunk<0x577380>::Call<void>(pointer);}       \
-                        void operator delete[](void * pointer)   { hook::StaticThunk<0x5773C0>::Call<void>(pointer);}
+#define ANGEL_ALLOCATOR void* operator new(size_t size)                        { return hook::StaticThunk<0x577360>::Call<void*>(size); } \
```

Comment:
Please add `void* operator new(size_t size, void * data) { return data; } \` for the `ANGEL_ALLOCATOR` macro as well

Reply:
Roger that!